nextflow.config

```diff
@@ -132,7 +132,7 @@ trace {
   file = "${params.tracedir}/execution_trace.txt"
 }
 dag {
-  enabled = true
+  enabled = false //requires graphic devices to run on ngp
```

Comment:
DAG can still be pushed to file and sent that way

Reply:
It failed while running the pipeline in NGP server. :(